tests/resources/testdistros/distro/usrlibosreleaseonly/usr/lib/os-release

```diff
@@ -0,0 +1,3 @@
+NAME="usr_lib_os-release_only"
```

Comment:
Can we have a real-life `/usr/lib/os-release` example if one is available?

Reply:
It would actually make the in-test data explain itself less. My vote against that, the test data seems perfectly fine to me.